### PR TITLE
Update Agent Integrations developer setup page

### PR DESCRIPTION
**Pull request by ofek**
### What does this PR do? What is the motivation?

Update docs to loosely match https://datadoghq.dev/integrations-core/setup/#ddev

### CODE REVIEW

File: content/en/developers/integrations/python.md [hunk 1]
```diff
@@ -78,110 +68,99 @@ For Linux installations, avoid modifying your system Python. Datadog recommends
...
 
 {{< tabs >}}
 {{% tab "MacOS" %}}
-1. Install pipx:
-   ```
-   brew install pipx
+1. In your browser, download the `.pkg` file: [ddev-5.2.1.pkg](https://github.com/DataDog/integrations-core/releases/download/ddev-v5.2.1/ddev-5.2.1.pkg)
+2. Run your downloaded file and follow the on-screen instructions.
+3. Restart your terminal.
+4. To verify that the shell can find and run the `ddev` command in your `PATH`, use the following command.
```
Comment (hestonhoffman) [inline content/en/developers/integrations/python.md:84]:
```suggestion
4. To verify that the `ddev` command has been added to your `PATH`, run the following command to retrieve the `ddev` version:
```

File: content/en/developers/integrations/python.md [hunk 2]
```diff
@@ -78,110 +68,99 @@ For Linux installations, avoid modifying your system Python. Datadog recommends
...
-
-{{% /tab %}}
-
-{{% tab "Linux" %}}
-1. Install pipx:
-   ```
-   python -m pip install pipx
-   ```
-1. Verify that pipx is installed:
+1. In your browser, download one the `.msi` files:
```
Comment (hestonhoffman) [inline content/en/developers/integrations/python.md:92]:
```suggestion
1. In your browser, download one of the following `.msi` files:
```

File: content/en/developers/integrations/python.md [hunk 3]
```diff
@@ -78,110 +68,99 @@ For Linux installations, avoid modifying your system Python. Datadog recommends
...
+     - [ddev-5.2.1-x64.msi](https://github.com/DataDog/integrations-core/releases/download/ddev-v5.2.1/ddev-5.2.1-x64.msi)
+     - [ddev-5.2.1-x86.msi](https://github.com/DataDog/integrations-core/releases/download/ddev-v5.2.1/ddev-5.2.1-x86.msi)
```
Comment (hestonhoffman) [inline content/en/developers/integrations/python.md:94]:
```suggestion
     - [ddev-5.2.1-x64.msi (64-bit)](https://github.com/DataDog/integrations-core/releases/download/ddev-v5.2.1/ddev-5.2.1-x64.msi)
     - [ddev-5.2.1-x86.msi (32-bit) ](https://github.com/DataDog/integrations-core/releases/download/ddev-v5.2.1/ddev-5.2.1-x86.msi)
```

File: content/en/developers/integrations/python.md [hunk 4]
```diff
@@ -78,110 +68,99 @@ For Linux installations, avoid modifying your system Python. Datadog recommends
...
+2. Run your downloaded file and follow the on-screen instructions.
+3. Restart your terminal.
+4. To verify that the shell can find and run the `ddev` command in your `PATH`, use the following command.
```
Comment (hestonhoffman) [inline content/en/developers/integrations/python.md:97]:
```suggestion
4. To verify that the `ddev` command has been added to your `PATH`, run the following command to retrieve the `ddev` version:
```

File: content/en/developers/integrations/python.md [hunk 5]
```diff
@@ -78,110 +68,99 @@ For Linux installations, avoid modifying your system Python. Datadog recommends
...
    ```
-   pipx --version
+   $ ddev --version
+   5.2.1
    ```
 {{% /tab %}}
 {{< /tabs >}}
 
-## Install the Datadog Agent Integration Developer Tool
+#### Install - command line
```
Comment (hestonhoffman) [inline content/en/developers/integrations/python.md:105]:
```suggestion
### Install from the command line
```

File: content/en/developers/integrations/python.md [hunk 6]
```diff
@@ -78,110 +68,99 @@ For Linux installations, avoid modifying your system Python. Datadog recommends
...
 
 {{< tabs >}}
 {{% tab "MacOS" %}}
-
-1. Run the following command and remove any executables shown in the output:
+1. Download the file using the `curl` command. The `-o` option specifies the file name that the downloaded package is written to. In this example, the file is written to `ddev-5.2.1.pkg` in the current directory.
+   ```
```
Comment (hestonhoffman) [inline content/en/developers/integrations/python.md:110]:
~~~suggestion
   ```shell
~~~

File: content/en/developers/integrations/python.md [hunk 7]
```diff
@@ -78,110 +68,99 @@ For Linux installations, avoid modifying your system Python. Datadog recommends
...
-   brew install pipx
+1. In your browser, download the `.pkg` file: [ddev-5.2.1.pkg](https://github.com/DataDog/integrations-core/releases/download/ddev-v5.2.1/ddev-5.2.1.pkg)
+2. Run your downloaded file and follow the on-screen instructions.
+3. Restart your terminal.
+4. To verify that the shell can find and run the `ddev` command in your `PATH`, use the following command.
    ```
-1. Check the Homebrew installation output and run any additional commands recommended by the installation script.
-
-1. Verify that pipx is installed:
+   $ ddev --version
```
Comment (hestonhoffman) [inline content/en/developers/integrations/python.md:86]:
```suggestion
   ddev --version
```

File: content/en/developers/integrations/python.md [hunk 8]
```diff
@@ -78,110 +68,99 @@ For Linux installations, avoid modifying your system Python. Datadog recommends
...
 {{< tabs >}}
 {{% tab "MacOS" %}}
-1. Install pipx:
-   ```
-   brew install pipx
+1. In your browser, download the `.pkg` file: [ddev-5.2.1.pkg](https://github.com/DataDog/integrations-core/releases/download/ddev-v5.2.1/ddev-5.2.1.pkg)
+2. Run your downloaded file and follow the on-screen instructions.
+3. Restart your terminal.
+4. To verify that the shell can find and run the `ddev` command in your `PATH`, use the following command.
    ```
```
Comment (hestonhoffman) [inline content/en/developers/integrations/python.md:85]:
~~~suggestion
   ```shell
~~~
This adds the copy button and `$`

File: content/en/developers/integrations/python.md [hunk 9]
```diff
@@ -78,110 +68,99 @@ For Linux installations, avoid modifying your system Python. Datadog recommends
...
-1. Verify that pipx is installed:
+1. In your browser, download one the `.msi` files:
+     - [ddev-5.2.1-x64.msi](https://github.com/DataDog/integrations-core/releases/download/ddev-v5.2.1/ddev-5.2.1-x64.msi)
+     - [ddev-5.2.1-x86.msi](https://github.com/DataDog/integrations-core/releases/download/ddev-v5.2.1/ddev-5.2.1-x86.msi)
+2. Run your downloaded file and follow the on-screen instructions.
+3. Restart your terminal.
+4. To verify that the shell can find and run the `ddev` command in your `PATH`, use the following command.
    ```
-   pipx --version
+   $ ddev --version
```
Comment (hestonhoffman) [inline content/en/developers/integrations/python.md:99]:
```suggestion
   ddev --version
```

File: content/en/developers/integrations/python.md [hunk 10]
```diff
@@ -78,110 +68,99 @@ For Linux installations, avoid modifying your system Python. Datadog recommends
...
-   python -m pip install pipx
-   ```
-1. Verify that pipx is installed:
+1. In your browser, download one the `.msi` files:
+     - [ddev-5.2.1-x64.msi](https://github.com/DataDog/integrations-core/releases/download/ddev-v5.2.1/ddev-5.2.1-x64.msi)
+     - [ddev-5.2.1-x86.msi](https://github.com/DataDog/integrations-core/releases/download/ddev-v5.2.1/ddev-5.2.1-x86.msi)
+2. Run your downloaded file and follow the on-screen instructions.
+3. Restart your terminal.
+4. To verify that the shell can find and run the `ddev` command in your `PATH`, use the following command.
    ```
```
Comment (hestonhoffman) [inline content/en/developers/integrations/python.md:98]:
~~~suggestion
   ```shell
~~~

File: content/en/developers/integrations/python.md [hunk 11]
```diff
@@ -78,110 +68,99 @@ For Linux installations, avoid modifying your system Python. Datadog recommends
...
 
 {{< tabs >}}
 {{% tab "MacOS" %}}
-
-1. Run the following command and remove any executables shown in the output:
+1. Download the file using the `curl` command. The `-o` option specifies the file name that the downloaded package is written to. In this example, the file is written to `ddev-5.2.1.pkg` in the current directory.
+   ```
+   curl -o ddev-5.2.1.pkg https://github.com/DataDog/integrations-core/releases/download/ddev-v5.2.1/ddev-5.2.1.pkg
+   ```
+2. Run the standard macOS [`installer`](https://ss64.com/osx/installer.html) program, specifying the downloaded `.pkg` file as the source. Use the `-pkg` parameter to specify the name of the package to install, and the `-target /` parameter for the drive in which to install the package. The files are installed to `/usr/local/ddev`, and an entry is created at `/etc/paths.d/ddev` that instructs shells to add the `/usr/local/ddev` directory to. You must include sudo on the command to grant write permissions to those folders.
```
Comment (hestonhoffman) [inline content/en/developers/integrations/python.md:113]:
```suggestion
2. Run the standard macOS [`installer`](https://ss64.com/osx/installer.html) program, specifying the downloaded `.pkg` file as the source. Use the `-pkg` parameter to specify the name of the package to install, and the `-target /` parameter for the drive in which to install the package. The files are installed to `/usr/local/ddev`, and an entry is created at `/etc/paths.d/ddev` that instructs shells to add the `/usr/local/ddev` directory to. You must include `sudo` on the command to grant write permissions to those folders.
```

File: content/en/developers/integrations/python.md [hunk 12]
```diff
@@ -78,110 +68,99 @@ For Linux installations, avoid modifying your system Python. Datadog recommends
...
+   ```
```
Comment (hestonhoffman) [inline content/en/developers/integrations/python.md:114]:
~~~suggestion
   ```shell
~~~

File: content/en/developers/integrations/python.md [hunk 13]
```diff
@@ -78,110 +68,99 @@ For Linux installations, avoid modifying your system Python. Datadog recommends
...
+   sudo installer -pkg ./ddev-5.2.1.pkg -target /
    ```
-   which -a ddev
+3. Restart your terminal.
+4. To verify that the shell can find and run the `ddev` command in your `PATH`, use the following command.
    ```
+   $ ddev --version
```
Comment (hestonhoffman) [inline content/en/developers/integrations/python.md:120]:
~~~suggestion
   ```shell
   ddev --version
~~~

File: content/en/developers/integrations/python.md [hunk 14]
```diff
@@ -78,110 +68,99 @@ For Linux installations, avoid modifying your system Python. Datadog recommends
...
+   5.2.1
+   ```
+{{% /tab %}}
 
-1. Make sure there are no virtual environments running:
-   1. Run the following command:
+{{% tab "Windows" %}}
+1. Download and run the installer using the standard Windows [`msiexec`](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/msiexec) program, specifying one of the `.msi` files as the source. Use the `/passive` and `/i` parameters to request an unattended, normal installation.
+   - `x64`:
       ```
```
Comment (hestonhoffman) [inline content/en/developers/integrations/python.md:128]:
~~~suggestion
      ```shell
~~~

File: content/en/developers/integrations/python.md [hunk 15]
```diff
@@ -78,110 +68,99 @@ For Linux installations, avoid modifying your system Python. Datadog recommends
...
-      echo $VIRTUAL_ENV
+      msiexec /passive /i https://github.com/DataDog/integrations-core/releases/download/ddev-v5.2.1/ddev-5.2.1-x64.msi
       ```
```
Comment (hestonhoffman) [inline content/en/developers/integrations/python.md:130]:
~~~suggestion
      ```shell
~~~

File: content/en/developers/integrations/python.md [hunk 16]
```diff
@@ -78,110 +68,99 @@ For Linux installations, avoid modifying your system Python. Datadog recommends
...
+   - `x86`:
+      ```
+      msiexec /passive /i https://github.com/DataDog/integrations-core/releases/download/ddev-v5.2.1/ddev-5.2.1-x86.msi
+      ```
+2. Restart your terminal.
+3. To verify that the shell can find and run the `ddev` command in your `PATH`, use the following command.
+   ```
+   $ ddev --version
+   5.2.1
+   ```
```
Comment (hestonhoffman) [inline content/en/developers/integrations/python.md:140]:
~~~suggestion
   ```shell
   ddev --version
   5.2.1
   ```
~~~

File: content/en/developers/integrations/python.md [hunk 17]
```diff
@@ -78,110 +68,99 @@ For Linux installations, avoid modifying your system Python. Datadog recommends
...
-     ```
-
-   - MacOS on Intel machines:
-     ```
-     pipx install --python /usr/local/bin/python3.9 ddev
-     ```
+### Standalone binaries
 
-1. Check the installation output and run any additional commands recommended by the installation script.
+After downloading the archive corresponding to your platform and architecture, extract the binary to a directory that is on your PATH and rename to `ddev`.
```
Comment (hestonhoffman) [inline content/en/developers/integrations/python.md:146]:
```suggestion
After downloading the archive corresponding to your platform and architecture, extract the binary to a directory that is on your `PATH` and rename the binary to `ddev`.
```

File: content/en/developers/integrations/python.md [hunk 18]
```diff
@@ -78,110 +68,99 @@ For Linux installations, avoid modifying your system Python. Datadog recommends
...
-   - ARM (M1+) machines:
-     ```
-     pipx install --python /opt/homebrew/bin/python3.9 ddev
-     ```
-
-   - MacOS on Intel machines:
-     ```
-     pipx install --python /usr/local/bin/python3.9 ddev
-     ```
+### Standalone binaries
```
Comment (hestonhoffman) [inline content/en/developers/integrations/python.md:144]:
```suggestion
### Install from a standalone binary
```

**Review by hestonhoffman — COMMENTED** [review]
Looks good - left some suggestions

File: content/en/developers/integrations/python.md [hunk 19]
```diff
@@ -78,110 +68,98 @@ For Linux installations, avoid modifying your system Python. Datadog recommends
...
+{{% /tab %}}
 
-1. Make sure there are no virtual environments running:
-   1. Run the following command:
+{{% tab "Windows" %}}
+1. Download and run the installer using the standard Windows [`msiexec`](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/msiexec) program, specifying one of the `.msi` files as the source. Use the `/passive` and `/i` parameters to request an unattended, normal installation.
+   - `x64`:
+      ```shell
+      msiexec /passive /i https://github.com/DataDog/integrations-core/releases/download/ddev-v5.2.1/ddev-5.2.1-x64.msi
+      ```shell
```
Comment (hestonhoffman) [inline content/en/developers/integrations/python.md:129]:
~~~suggestion
      ```
~~~
Oops, wrong one

File: content/en/developers/integrations/python.md [hunk 20]
```diff
@@ -78,110 +68,98 @@ For Linux installations, avoid modifying your system Python. Datadog recommends
...
+   - `x86`:
       ```
```
Comment (hestonhoffman) [inline content/en/developers/integrations/python.md:131]:
~~~suggestion
      ```shell
~~~

**Review by hestonhoffman — APPROVED** [review]

**Review by FlorentClarret — APPROVED** [review]

File: content/en/developers/integrations/python.md [hunk 21]
```diff
@@ -78,110 +68,98 @@ For Linux installations, avoid modifying your system Python. Datadog recommends
...
 {{< tabs >}}
 {{% tab "MacOS" %}}
-1. Install pipx:
-   ```
-   brew install pipx
-   ```
-1. Check the Homebrew installation output and run any additional commands recommended by the installation script.
-
-1. Verify that pipx is installed:
+1. In your browser, download the `.pkg` file: [ddev-5.2.1.pkg](https://github.com/DataDog/integrations-core/releases/download/ddev-v5.2.1/ddev-5.2.1.pkg)
```
Comment (vivek-datadog) [inline content/en/developers/integrations/python.md:80]: If the version is not auto updated, let's add a note stating to use the required version of ddev and 5.2.1 is provided as an example in all the referenced places.

File: content/en/developers/integrations/python.md [hunk 22]
```diff
@@ -3,21 +3,11 @@ kind: documentation
...
-- Windows AMD64 (64-bit) MSI installer
-- Windows x86 (32-bit) MSI installer
-- Windows universal (AMD64+x86) EXE installer
-- macOS DMG installer
+This document covers how to set up a Python environment to work on Agent-based Integrations, including installing the interpreter and ensuring all of the required dependencies are present. Alternatively, you can use the standalone binaries and installers provided with each release.
 
 ## Install Python
 
-Many operating systems come with a pre-installed version of Python. However, the version of Python installed by default may be older than the version used in the Agent, and may lack some required tools and dependencies. To ensure that you've everything you need to get an integration running, install a dedicated Python interpreter.
+Many operating systems come with a pre-installed version of Python. However, the version of Python installed by default may be older than the version used in the Agent, and may lack some required tools and dependencies. To ensure that you have everything you need to get an integration running, install a dedicated Python interpreter.
```
Comment (vivek-datadog) [inline content/en/developers/integrations/python.md:10]: Rephrase to `However, Python version installed by default may be older than the version used in the Agent, and may lack some required tools and dependencies.` to limit sentence with 25 words or fewer.

File: content/en/developers/integrations/python.md [hunk 23]
```diff
@@ -3,21 +3,11 @@ kind: documentation
...
-## GitHub releases
-
-Each [release](https://github.com/DataDog/integrations-core/releases?q=ddev-&expanded=true) provides the following:
-
-- Standalone binaries for Linux, Windows, and macOS
-- Windows AMD64 (64-bit) MSI installer
-- Windows x86 (32-bit) MSI installer
-- Windows universal (AMD64+x86) EXE installer
-- macOS DMG installer
+This document covers how to set up a Python environment to work on Agent-based Integrations, including installing the interpreter and ensuring all of the required dependencies are present. Alternatively, you can use the standalone binaries and installers provided with each release.
```
Comment (vivek-datadog) [inline content/en/developers/integrations/python.md:6]: Rephrase to `This document covers setting up Python environment with all the required dependencies to work on Agent-based Integrations. Alternatively, you can directly use the standalone binaries and installers provided with each release.`

**Review by vivek-datadog — CHANGES_REQUESTED** [review]

File: content/en/developers/integrations/python.md (hunk 21 above)
Comment (ofek) [reply content/en/developers/integrations/python.md:80]: The version will be dynamically rendered in an upcoming PR

**Pull request merged**